backend/eval.py

```diff
@@ -19,21 +19,35 @@ async def generate_code_core(image_url: str, stack: str) -> str:
     prompt_messages = assemble_prompt(image_url, stack)
     openai_api_key = os.environ.get("OPENAI_API_KEY")
     openai_base_url = None
+    azure_openai_api_key = os.environ.get("AZURE_OPENAI_API_KEY")
+    azure_openai_resource_name = os.environ.get("AZURE_OPENAI_RESOURCE_NAME")
+    azure_openai_deployment_name = os.environ.get("AZURE_OPENAI_DEPLOYMENT_NAME")
+    azure_openai_api_version = os.environ.get("AZURE_OPENAI_API_VERSION")
 
     pprint_prompt(prompt_messages)
 
     async def process_chunk(content: str):
         pass
 
+    if not openai_api_key and not azure_openai_api_key:
+        raise Exception("OpenAI API or Azure key not found")
+
     if not openai_api_key:
```

Comment:
shouldn't you swap the condition with azure here?

Reply:
i will check if there is one of two, is missing all raise exception (but maybe is better have 2 separate checks)

Reply:
I'm going to rework this a bit today before merging.

Reply:
Thank you!

Reply:
Thank you for this PR! Excited to get it in.